Samples/MobileBuyIntegration/MobileBuyIntegration/CartManager.swift

```diff
@@ -21,13 +21,14 @@
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import Buy
+@preconcurrency import Buy
 import Combine
 import Foundation
 import PassKit
 import ShopifyCheckoutSheetKit
 
 // swiftlint:disable type_body_length
+@MainActor
```

Comment:
The cart manager deals with a UI-related cart, so we can mark it as a MainActor so that everything happens on the main thread